### PR TITLE
feat(wip): Billing Page

### LINKED ISSUE #326: Define Billing Page API contract
Currently the Billing page is server rendered and served by Quartz.  This contract will define all the endpoints needed for `influxdata/ui` to fetch the required state to render and use the page.

### PR DESCRIPTION
**Pull request by 121watts**
Closes #326 

One question for me is how to get the current user's email.  Seems odd to have an entire endpoint just for this one string.  Turns out there was a bunch of information on the `Account` struct that we don't use for the billing page at all.

### CODE REVIEW

File: src/client/unity.yml [hunk 1]
```diff
@@ -5,6 +5,187 @@ info:
...
+      requestBody:
+        description: Billing Contact to Update
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/BillingNotifySettings"
+      responses:
+        "200":
+          description: billing contact updated
```
Comment (randycoulman) [inline src/client/unity.yml:57]: Looks like some copy-pasta in this `put` definition; several references to billing contact and/or PaymentMethods, but this is about notification settings.

File: src/client/unity.yml [hunk 2]
```diff
@@ -5,6 +5,187 @@ info:
...
+              schema:
+                $ref: "#/components/schemas/Error"
+  "/billing/contact":
+    get:
+      operationId: GetBillingContact
+      tags:
+        - BillingContact
+      responses:
+        "200":
+          description: A Credit Card parameter object
```
Comment (randycoulman) [inline src/client/unity.yml:75]: copy-pasta?

File: src/client/unity.yml [hunk 3]
```diff
@@ -5,6 +5,187 @@ info:
...
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+    put:
+      operationId: PutBillingContact
+      tags:
+        - PaymentMethods
```
Comment (randycoulman) [inline src/client/unity.yml:89]: copy-pasta?

File: src/client/unity.yml [hunk 4]
```diff
@@ -5,6 +5,187 @@ info:
...
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+  "/billing/payment_method":
+    put:
+      operationId: PutPaymentMethod
+      tags:
+        - PaymentMethods
+      requestBody:
+        description: Invite to be sent
```
Comment (randycoulman) [inline src/client/unity.yml:152]: copy-pasta?

**Review by randycoulman — CHANGES_REQUESTED** [review]
LGTM, but there's some copy-pasta in the endpoint descriptions to clean up.

For getting the user's e-mail, should we have a more generic return type for the `/billing/account` endpoint that includes the email in addition to the few account fields we need?

**Review by drdelambre — APPROVED** [review]

**Comment by 121watts** [comment]
@randycoulman i fixed the copy pasta stuff and added email to the `Account` struct.

**Review by randycoulman — APPROVED** [review]

**Review by Palakp41 — APPROVED** [review]
👍

**Pull request merged**